the-collab-lab/tcl-22-smart-shopping-list
# Share your token styling

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

Adding styling to the Share Your Component view on mobile. 

## Related Issue

#14 

## Acceptance Criteria

* As a user, I want the app to look professional and welcoming



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :nail_care: Style              |

## Updates

### Before

![Screen Shot 2021-05-29 at 9 22 28 AM](https://user-images.githubusercontent.com/59854488/120071998-99d17800-c05f-11eb-827c-31345d38bf3a.png)


### After

![Screen Shot 2021-05-29 at 9 26 46 AM](https://user-images.githubusercontent.com/59854488/120072102-02b8f000-c060-11eb-9d0f-f5069ffa23fc.png)


## Testing Steps / QA Criteria

1. Follow the [preview link](https://tcl-22-shopping-list--pr68-shareyourtoken-styli-rfek8i4g.web.app)
2. Launch the app locally from your terminal:
- Pull down this branch with `git pull origin shareYourToken-styling` and check that branch out with `git checkout origin shareYourToken-styling`
- Run `npm install` to update dependencies, then `npm start` to launch the app in your browser at http://localhost:3000

In the app, navigate to the Share Your Token view to see the changes.
